modules/client/src/interfaces.ts

```diff
@@ -553,20 +555,57 @@ export type SubgraphPluginRepoListItem = {
   releases: SubgraphPluginRepoReleaseListItem[];
 };
 
+export type SubgraphPluginVersion = {
+  build: number;
+  metadata: string;
+  release: {
+    release: number;
+    metadata: string;
+  };
+  pluginRepo: {
+    id: string;
+    subdomain: string;
+  };
+};
+
 export type PluginRepoReleaseMetadata = {
   name: string;
   description: string;
   images: Object; // TODO specify parameters
 };
+export type PluginRepoBuildMetadata = {
+  ui: string;
+  change: string;
+  pluginSetupABI: Object;
+};
 
 export type PluginRepoRelease = {
   release: number;
   metadata: PluginRepoReleaseMetadata;
-  builds: number[];
+  latestBuild: number;
 };
 
-export type PluginRepo = {
+export type PluginRepoListItem = {
   address: string;
   subdomain: string;
   releases: PluginRepoRelease[];
 };
+
+export type PluginRepo = {
```

Comment:
Not sure I follow. A plugin repo contains a list of releases. Each release contains a list of builds. How can the plugin repo contain a release and build at the root?

If we are providing the latest version for convenience, then let's make it explicit:

```ts
  current: {
    release: {
      number,
      metadata
    },
    build: {
      number
      metadata
    }
  }
```